pkg/minikube/bootstrapper/bsutil/binaries.go

```diff
@@ -37,6 +37,23 @@ import (
 	"k8s.io/minikube/pkg/minikube/vmpath"
 )
 
+// TransferCrictl transfers the crictl binary to the machine to replace /usr/bin/crictl
+func TransferCrictl(cfg config.KubernetesConfig, c command.Runner, crictlVersion string) error {
+	src, err := download.CrictlBinary(cfg.KubernetesVersion, crictlVersion)
+	if err != nil {
+		if strings.Contains(err.Error(), "response code: 404") {
+			klog.Warning("Failed to download crictl with matching version. Using whatever version is already on the image")
```

Comment:
```suggestion
			klog.Errorf("Failed to download crictl %s. Will attempt to continue using the preinstalled version.", crictlVersion)
```
We should surface this to the user via `Error` since there's a good chance once we upgrade the crictl version the start will fail.